src/components/FromSTIX/CustomFieldModal.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { Modal, TextInput } from "carbon-components-react";
+import { useDispatch, useSelector } from "react-redux";
+import { closeCustomFieldModal, addField } from "../../store/actions/from_stix";
+
+const CustomFieldModal = () => {
+  const modalShow = useSelector((state) => state.fromStix.modalShow);
+  const dispatch = useDispatch();
+  const [customField, setCustomField] = useState("");
+  const isValidCustomField = (customField) => {
```

Comment:
can be exported to `global` folder as `isValidCustomStixField` to use in future components, can create stixHelper.js file in global folder